app/models/tube.rb

```diff
@@ -40,12 +40,6 @@ def pool_id
     submissions.ids.first
   end
 
-  def ancestor_of_purpose(ancestor_purpose_id)
-    return self if plate_purpose_id == ancestor_purpose_id
-
-    ancestors.order(created_at: :desc).find_by(plate_purpose_id: ancestor_purpose_id)
```

Comment:
I expect you noticed but just pointing out that this one uses 'created_at' rather than 'id' to order. But we have made a lot of changes recently in this direction, so changing it is probably a good thing?

Reply:
Oh, nice catch. I hadn't actually noticed. I did actually eyeball them to make sure they were the same, but obviously missed that.

You're right though, this should be a good thing.